apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/lru/LruNormalizedCache.java

```diff
@@ -70,8 +86,8 @@ public LruNormalizedCache(EvictionPolicy evictionPolicy, NormalizedCache seconda
   }
 
   @Nonnull @Override public Set<String> merge(Record apolloRecord) {
-    if (secondaryCacheStore.isPresent()) {
-      secondaryCacheStore.get().merge(apolloRecord);
+    if (secondaryCache.isPresent()) {
```

Comment:
I wish we had selected Java8 Optionals as internal.
this if can be replaced with Java8 optional:
```
secondaryCache.ifPresent(it -> it.merge(apolloRecord))
```